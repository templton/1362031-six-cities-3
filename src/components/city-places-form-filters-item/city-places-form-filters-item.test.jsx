import React from "react";
import renderer from "react-test-renderer";
import CityPlacesFormFiltersItem from "./city-places-form-filters-item";

describe(`Render <CityPlacesFormFiltersItem/>`, () => {
  it(`Render page`, () => {
    const tree = renderer
      .create(<CityPlacesFormFiltersItem item={``} className={``} onClick={()=>{}}/>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

