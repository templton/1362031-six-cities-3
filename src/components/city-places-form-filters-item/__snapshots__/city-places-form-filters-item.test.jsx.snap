// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render <CityPlacesFormFiltersItem/> Render page 1`] = `
<li
  className=""
  onClick={[Function]}
>
  
</li>
`;
